### DISCUSSION TITLE
Why is there (os error 2) when trying to publish move contract

### DISCUSSION THREAD

**Discussion by sidharthpunathil** [aptos-labs/aptos-developer-discussions#465] [Questions]
### Discord user ID

callmexid

### Describe your question in detail.

I am trying to do 
`aptos move publish`

### What error, if any, are you getting?

```
{
  "Error": "Unexpected error: The system cannot find the file specified. (os error 2)"
}
```

![image](https://github.com/user-attachments/assets/ceac6abe-ad02-43c2-a81c-9d6f0a350d15)

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

### Describe your environment or tooling in detail

Move.toml 

```
[package]
name = "HeartProtocolV2"
version = "1.0.0"
authors = ["Sidharth P"]

[addresses]
heartprotocol = "0xfd5601d516d9b5cded773da4e9dad13c829a0a96677498e64a0efa683dcceb52"

[dev-addresses]
heartprotocol = "0xfd5601d516d9b5cded773da4e9dad13c829a0a96677498e64a0efa683dcceb52"

[dependencies]
AptosFramework = { git = "https://github.com/aptos-labs/aptos-core.git", subdir = "aptos-move/framework/aptos-framework", rev = "testnet" }
```

**Comment by 0x-j** [comment]
This is weird, can you share your code and I'll try to publish your code from my laptop?
